src/iter.rs

```diff
@@ -222,6 +225,19 @@ pub trait Reads: Send + Sync {
         )
     }
 
+    #[must_use]
+    fn norm(
+        self,
+        selector_expr: SelectorExpr,
+        label: Label,
+        range: (usize, usize),
```

Comment:
Use a generic bounded by [RangeBounds](https://doc.rust-lang.org/std/ops/trait.RangeBounds.html) for Rust's `1..3` syntax! See the `length_in_bounds` operation for my usage.

Reply:
I want to be able to get both of the two number separately. Don't think there is convenient way to do that with RangeBounds. Let me know if I'm wrong.

Reply:
Does [this](https://doc.rust-lang.org/std/ops/trait.RangeBounds.html#tymethod.start_bound) work?

Reply:
This is what I saw but requires unpacking from the Bound. If you think this is cleaner let me know.

Reply:
Yeah its annoying to unpack, but I would prefer to have the nice `..` syntax for the API.